ncrypt/crypt.c

```diff
@@ -1085,7 +1085,7 @@ static void crypt_fetch_signatures(struct Body ***b_sigs, struct Body *b, int *n
     else
     {
       if ((*n % 5) == 0)
-        mutt_mem_realloc(b_sigs, (*n + 6) * sizeof(struct Body **));
+        MUTT_MEM_REALLOC(b_sigs, *n + 6, struct Body *);
 
       (*b_sigs)[(*n)++] = b;
     }
```

Comment:
This is a bugfix.  I'll fix it in a separate commit.  Now I realize why I was having unexpected compiler errors here until I removed a `*`: it was wrong before me touching it.

Reply:
@gahr @ossilator 

If I hadn't added the type to the realloc macro, this bug would have passed inadvertently.

You could argue that it would have been fixed anyway when switching to the macro, but I prefer a loud fix than a silent fix.